tachyon/math/finite_fields/generator/prime_field_generator/config.h.tpl

```diff
@@ -51,15 +51,14 @@ class TACHYON_EXPORT %{class}Config {
     %{one_mont_form}
   });
 
-  constexpr static BigInt<%{n}> kSubgroupGenerator = BigInt<%{n}>({
-    %{subgroup_generator}
-  });
-
   constexpr static bool kHasTwoAdicRootOfUnity = %{has_two_adic_root_of_unity};
 
   constexpr static bool kHasLargeSubgroupRootOfUnity = %{has_large_subgroup_root_of_unity};
 
 %{if kHasTwoAdicRootOfUnity}
+  constexpr static BigInt<%{n}> kSubgroupGenerator = BigInt<%{n}>({
+    %{subgroup_generator}
+  });
```

Comment:
All prime fields have `kSubgroupGenerator` which generates every element of the field.
As i know, a generator which generates every element of 2-adic-group is called `kTwoAdicRootOfUnity` in our prime field config.

Reply:
Therefore this commit should be reverted.

Reply:
I don't think so. I believe this value is only needed by fft friendly field